inc/toolbox.class.php

```diff
@@ -508,7 +508,7 @@ static function logInFile($name, $text, $force = false) {
     * @param $linenum   integer  line number the error was raised at.
     * @param $vars      array    that points to the active symbol table at the point the error occurred.
    **/
-   static function userErrorHandlerNormal($errno, $errmsg, $filename, $linenum, $vars) {
+   static function userErrorHandlerNormal($errno, $errmsg, $filename, $linenum) {
```

Comment:
Since this method is called from `userErrorHandlerDebug`; you should change this one too.